app/ai/voice/agents/breeze_buddy/managers/utils.py

```
@@ -14,12 +14,81 @@
+                            resolved_value = func(**fn_params) if fn_params else func()
+                        else:
+                            resolved_value = (
+                                func(resolved_value, **fn_params)
+                                if fn_params
+                                else func(resolved_value)
+                            )
+                        logger.info(
+                            f"Applied function '{fn_name}' to field '{key}', "
+                            f"result: '{resolved_value}'"
+                        )
+                    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)